ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/BlockImportResult.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.mainnet;
+
+public class BlockImportResult {
+
+  public enum BlockImportStatus {
+    TRUE,
+    FALSE,
```

Comment:
so TRUE is "freshly imported" and FALSE is "invalid block"? do we need that nuance in the result options?

Reply:
Yes, that is correct. Happy to change the names to something more meaningful.

Reply:
Replaced the values with more meaningful names